mw-webapp/src/utils/auth/handleUserCredentials.ts

```diff
@@ -0,0 +1,18 @@
+import {getRedirectResult} from "firebase/auth";
+import {writeNewUserData} from "src/utils/auth/writeNewUserData";
+import {auth} from "src/firebase";
+
+export const handleUserCredentials = async () => {
+  try {
+    const userCredentials = await getRedirectResult(auth);
```

Comment:
i dont understand why network related methods are in the utils

Reply:
@Ekaterina1994

Reply:
I will move it to the appropriate folder if you tell me where it should be